src/Umbraco.Web/packages.config

```diff
@@ -26,7 +26,10 @@
   <package id="MiniProfiler" version="2.1.0" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net45" />
   <package id="Owin" version="1.0" targetFramework="net45" />
+  <package id="RestSharp" version="104.4.0" targetFramework="net45" />

```

Comment:
I assume RestSharp and ServiceStack.Text are part of the slack lib? (they should be removed too)